lib/client/node.go

```diff
@@ -115,6 +115,16 @@ func (h *nodes) Delete(metadata api.NodeMetadata) error {
 		}
 	}
 
+	// Remove Node host directory
+	log.Debug("Removing Node host data")
```

Comment:
same with this log - I think it means to be "Removing Node BGP configuration" or similar.